tests/fixtures.py

```diff
@@ -126,19 +126,31 @@ def specs_fixture_with_depends():
                             }})
     _write('service', 'servica', {'image': 'busybox'})
 
-def fixture_with_commands(once_fail=False, always_fail=False):
-    once_commands = ['touch /once_test_file', 'echo "once ran" | tee -a /once_test_file']
+def fixture_with_commands(once_fail=False, always_fail=False, test_fail=False):
+    once_commands = ['touch /once_test_file', 'stdbuf -i0 -o0 -e0 echo "once ran" | tee -a /once_test_file']
```

Comment:
http://stackoverflow.com/questions/11337041/force-line-buffering-of-stdout-when-piping-to-tee